packages/gestalt/src/Table.js

```diff
@@ -20,6 +20,7 @@ export default function Table(props: Props) {
   return (
     <Box
       overflow="scrollX"
+      paddingX={1}
```

Comment:
@PayalModi could we remove this? In Gestalt, every container element should we 100% width by default without any extra padding.